migrator/migrations/n_39_to_n_40_postgres_policies/legacy/store.go

```diff
@@ -24,6 +28,7 @@ type Store interface {
 // New returns a new Store instance using the provided bolt DB instance.
 func New(db *bolt.DB) Store {
 	bolthelper.RegisterBucketOrPanic(db, policyBucket)
+	bolthelper.RegisterBucketOrPanic(db, removedDefaultPolicyBucket)
```

Comment:
Does this bucket still exist after 66?  For instance if a customer started at 70, do we have the bucket just empty?

Reply:
Yes, it still exists. If a customer started at 70, it is just empty but we cannot assume that.